relayer/chain/relaychain/connection.go

```diff
@@ -221,58 +191,15 @@ func (co *Connection) FetchMMRLeafCount(relayBlockhash types.Hash) (uint64, erro
 	return mmrLeafCount, nil
 }
 
-func (co *Connection) fetchKeys(keyPrefix []byte, blockHash types.Hash) ([]types.StorageKey, error) {
```

Comment:
Seems just move position of this function, there is no internal change?

If so would suggest to leave it untouched to make diff a bit easier.

Reply:
addressed in https://github.com/Snowfork/snowbridge/pull/1288/commits/f6b090c7bcedde5f79d34eeacc4de80d21dc0614